datalad/support/gitrepo.py

```diff
@@ -54,6 +54,7 @@
 from datalad.utils import getpwd
 from datalad.utils import updated
 from datalad.utils import posix_relpath
+from datalad.utils import assert_no_open_files
```

Comment:
why the import?

Reply:
I guess left over from me trying to stick it int some "privates" (well -- the `__del__`) of GitRepo.  Now I forgot why I removed it from there, so will keep it at it and remove the import, thanks